src/WritePlasterContent.ps1

```diff
@@ -0,0 +1,246 @@
+function Write-PlasterContent {
+    <#
```

Comment:
Please follow the help comment style of the rest of the project.  Comment help goes outside and above the function.